base/audit_events.go

```diff
@@ -685,9 +702,13 @@ var AuditEvents = events{
 		Name:        "Update Inter-Sync Gateway Replication",
 		Description: "Inter-Sync Gateway Replication was updated",
 		MandatoryFields: AuditFields{
-			"db":             "database name",
-			"replication_id": "replication id",
-			"config":         "JSON representation of new replication config",
+			AuditFieldReplicationID: "replication id",
+			"config":                "JSON representation of new replication config",
```

Comment:
I found "new" to be a bit ambiguous here: since this endpoint supports upsert, "new" sounded like it could be the result of the merge (when it's actually, correctly, the incoming payload).   Should we use something like "payload" consistently for all events when referring to the body of the incoming request?

Reply:
Agreed, I think I want to change this everywhere, but let me know if this doesn't make sense.